server/routes/auth.js

```diff
@@ -1,3 +1,6 @@
+// TODO next() is a placeholder in case of some continuation of an endpoint in the future
```

Comment:
this would be more useful if you leave the comment right next to the line you use `next()`